### PR TITLE
Update var_info.unified

### PR DESCRIPTION
**Pull request by ProfessorKaos64**
- Match control.in.unified per old PR #23 and @wsnipex 's suggestion.

### CODE REVIEW

File: var_info.unified [hunk 1]
```diff
@@ -44,7 +44,7 @@ BUILD_DEPENDS="debhelper (>= 7.0.50~),
  libgles2-mesa-dev [armel] | libgl1-mesa-dev | libgl-dev,
  libglew-dev,
  libglu1-mesa-dev | libglu-dev,
- libgnutls-dev,
+ libgnutls-deb0-28 | libgnutls28 | libgnutls26,
```
Comment (wsnipex) [inline var_info.unified]: Don't drop libgnutls-dev. And also add it to build depends section of control.unified as you had on the old PR.

Comment (ProfessorKaos64) [inline var_info.unified]: Will do.

**Comment by ProfessorKaos64** [comment]
I rebased and reset my master branch fork as well for any future pulls.

**Comment by wsnipex** [comment]
please squash to a single commit.

**Comment by ProfessorKaos64** [comment]
I did a squash on the commits, but I was unsure if it went through correctly. I saw you made commit 0594b42. Is there still an issue with this PR, or are changes in already and this can be manually closed?

**Comment by ProfessorKaos64** [comment]
I squashed the commits I had, but now see commits at 2 due to the one you have. Is this acceptable?

**Comment by wsnipex** [comment]
this commit has actually been there for 1,5 months. Not sure what you based your branch on, but something was wrong on your end. Are you saying it works on debian with master as is?

**Comment by ProfessorKaos64** [comment]
I'm not sure what happened with the squash. To get a clean 1 commit, it would be easiest for me to resubmit. I apologize for the error on the squash. I actually appreciate your team having good standards for pulls, as well as being patient with me. I am still constantly learning collaboration on git.

If i resubmit, is [this commit](https://github.com/ProfessorKaos64/xbmc-packaging/commit/5ea13a371da3941ad2e11447b35bf5a67fbe9976) of the two attached acceptable for the PR? I will get that to you. I

**Comment by wsnipex** [comment]
as fritsch already explained before, you do not need to close your PR.
This is the proper workflow:
1. reset your branch on our master:
assuming you added xbmc-packaging as remote: git fetch xbmc-packaging; git rebase -i xbmc-packaging/master
2. during rebase make sure only your own commits are picked, then squash (s) all down to 1
3. git push -f origin yourbranch

**Comment by ProfessorKaos64** [comment]
I am going to add this as a note, since I already cleaned master and resubmitted. I keep a store of git advice and tips like this, so I'll be sure not to mess this up in the future @wsnipex . I did a similar set of actions before, but did not do them 100% correct since I somehow squashed and included that old commit.

**Comment by ProfessorKaos64** [comment]
Btw, would you do a hard reset or a soft reset?
```
git reset --hard origin/master
```

**Comment by wsnipex** [comment]
hard reset. But beware this will delete any local changes, so do it before adding your changes.

**Pull request closed without merging**